packages/common/src/WalletFactoryApi.ts

```diff
@@ -4,33 +4,33 @@ import { Contract } from 'ethers'
 
 import * as _ from 'lodash'
 
-export async function getInitializers(
-    initializerDto: InitializerDto
-  ): Promise<ISmartAccount> {    
-    const { chainId, owner,  txServiceUrl} = initializerDto
+export async function getWalletInfo(
+  initializerDto: InitializerDto
+): Promise<ISmartAccount> {
+  const { chainId, owner, txServiceUrl } = initializerDto
 
-    const smartAccountInfo = await new NodeClient({txServiceUrl}).getSmartAccountsByOwner({
-      owner: owner,
-      chainId
-    })
-    if (!smartAccountInfo.data || smartAccountInfo.data.length == 0){
-      throw new Error("No Smart Account Found against supplied EOA");      
-    }
-    // check wallet is deployed on not
-    let wallet = _.filter(smartAccountInfo.data, {chainId: chainId, 'isDeployed': true})
-    if (!wallet){
+  const smartAccountInfo = await new NodeClient({ txServiceUrl }).getSmartAccountsByOwner({
```

Comment:
what if node client API fails. there should be catch and possibly fallback to default querying methods